### ISSUE TITLE
lefthook.yml created in every directory 

### ISSUE THREAD

**Issue by vishal1132** [evilmartians/lefthook#221]
Hi, i have installed lefthook for mac via homebrew and then `lefthook install` in a particular directory where defined `pre-commit` hook for lefthook. Now whenever i commit in any repository in any directory, if the lefthook file is not present it's automatically created. I don't want lefthook to be a part of all my projects. How to resolve that?

**Comment by Envek** [comment]
This is very weird!

What is your git hooks path in your repositories? E.g. what `git rev-parse --git-path hooks` returns in different repos? It should be `.git/hooks` everywhere (so, every repo should have its own local set of hooks)

**Comment by vishal1132** [comment]
@Envek Oh, I have had global git hooks configured, is it because of that?

**Comment by Envek** [comment]
Yeah, exactly. Not sure how lefthook should behave in such situation.

**Comment by vishal1132** [comment]
@Envek I see, thanks for the reply. I guess I will close this, and remove the global githook and check and might open again if the issue persists?
